portfolio/src/main/webapp/script.js

```diff
@@ -39,8 +39,9 @@ function showSlides(n) {
 document.getElementById('movie-container').addEventListener("load", movieRecommender());
 window.addEventListener("load", getComments());
 document.getElementById('map').addEventListener("load", createMap());
-
-
+window.addEventListener('load', fetchUpload());
```

Comment:
Not sure if this was a merge issue but it was pointed out the issue that this is calling the function instead of setting it to the listener to be called later

Reply:
ah yes I fixed it on the other branch, so I'll keep that when merging these two branches